.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,30 @@
+name: pythontestrunner
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Building ptr on python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
```

Comment:
I only see Linux

Reply:
Yes, my bad!
I’ll add both windows and macos builds

Reply:
Fixed!